We want to install (not all of) those files manually.

--- a/Makefile.am	2017-05-24 23:59:15.000000000 +0200
+++ b/Makefile.am	2017-05-26 21:52:01.184258647 +0200
@@ -4,16 +4,12 @@
 
 AM_CPPFLAGS = -I$(top_srcdir)/include
 
-dist_doc_DATA = AUTHORS README.md
-nodist_doc_DATA = VERSION
 nodist_pkgconf_DATA = misc/mdds-@API_VERSION@.pc
 
 DISTCLEANFILES = misc/mdds-@API_VERSION@.pc
 
 EXTRA_DIST = \
 	autogen.sh \
-	CHANGELOG \
-	LICENSE \
 	doc/conf.py \
 	doc/doxygen.conf \
 	doc/flat_segment_tree.rst \
@@ -151,14 +147,6 @@
 	trie_map_test_mem.mem
 endif
 
-install-data-local:
-	$(MKDIR_P) $(DESTDIR)$(docdir)
-	$(INSTALL_DATA) $(top_srcdir)/LICENSE $(DESTDIR)$(docdir)/COPYING
-	$(INSTALL_DATA) $(top_srcdir)/CHANGELOG $(DESTDIR)$(docdir)/NEWS
-
-uninstall-local:
-	rm -f $(DESTDIR)$(docdir)/COPYING $(DESTDIR)$(docdir)/NEWS
-
 if BUILD_DOCS
 html-local:
 	(cd doc && $(DOXYGEN) doxygen.conf)
